examples/resources/opslevel_team/resource.tf

```diff
@@ -2,13 +2,18 @@ data "opslevel_group" "foo" {
   identifier = "foo"
 }
 
+data "opslevel_team" "parent" {
+  alias = "platform"
+}
+
 resource "opslevel_team" "example" {
   name             = "foo"
   manager_email    = "john.doe@example.com"
   members          = ["john.doe@example.com", "jane.doe@example.com"]
   responsibilities = "Responsible for foo frontend and backend"
   aliases          = ["bar", "baz"]
   group            = data.opslevel_group.foo.alias
```

Comment:
we should remove this now and also deprecate this field

```suggestion
```

Reply:
Can you also do a search through the examples if we use `opslevel_team` in any other examples that use the `group` field

Reply:
For sure 👍